src/main/resources/db/migration/V0_0_1__initial_migration.sql

```diff
@@ -589,7 +589,7 @@ CREATE TABLE IF NOT EXISTS location_relationship
     location_hierarchy_identifier UUID                     NOT NULL,
     location_identifier           UUID                     NOT NULL,
     parent_identifier             UUID,
-    ancestry                      UUID[]                   NOT NULL,
+    ancestry                      UUID[]                   ,
```

Comment:
how come ancestry is optional?

Reply:
We didn't use this column anywhere, so I was talking with Linda to maybe remove it from db, so first I delete not_null constraint to see is there going to be an exception if it is null.. so maybe we can remove it now